DetectorDescription/DDCMS/plugins/test/DDTestSpecParsFilter.cc

```diff
@@ -71,7 +66,7 @@ void DDTestSpecParsFilter::analyze(const Event&, const EventSetup& iEventSetup)
       for (const auto& kl : t.second->numpars) {
         log << kl.first << " = ";
         for (const auto& kil : kl.second) {
-          log << kil / MM_2_CM << " ";
+          log << kil << " ";
```

Comment:
This line outputs `TrackerXi` and `TrackerRadLength`, which are defined as plain numbers without units in XML. Their values as returned by DD4hep should not change with the DD4hep units change, so this PR removes the units conversion in the output.